src/app/components/osd/osd.component.ts

```diff
@@ -185,11 +189,14 @@ export class OsdComponent implements AfterViewInit, OnDestroy {
 
         this.viewer.addHandler('page', ({ page }) => {
           this.pageChange.next(page + 1);
+          this._page = page;
+          this.pageImg.emit(page);
```

Comment:
Here you could just use the `pageChange` event

Reply:
By simply using the `pageChange` event, it is not possible to select the image with index 0 (the front cover) because the page counter starts at 1.